test/server_testing_tools.h

```diff
@@ -54,10 +54,24 @@ class ZimFileServer
   typedef std::shared_ptr<httplib::Response>  Response;
   typedef std::vector<std::string> FilePathCollection;
 
+  enum Options
+  {
+    NO_TASKBAR_NO_LINK_BLOCKING = 0,
+    NO_TASKBAR                  = 0,
```

Comment:
Do we really need this `NO_TASKBAR` when it is the same thing than `NO_TASKBAR_NO_LINK_BLOCKING` ?

Reply:
This concern is dropped from the agenda due to `NO_TASKBAR` becoming unused and therefore not in any way justified.